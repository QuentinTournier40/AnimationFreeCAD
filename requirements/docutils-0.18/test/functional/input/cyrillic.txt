Заголовок
---------

первый пример: "Здравствуй, мир!"

Title
-----

.. class:: language-en

first example: "Hello world".

Примечания
----------

Этот документ испытивает изображение LaTeXом кириллических букв.

.. class:: language-en

This document tests the rendering of Latin and Cyrillic characters by the
LaTeX and XeTeX writers. Check the compiled PDF for garbage characters in
text and bookmarks.


