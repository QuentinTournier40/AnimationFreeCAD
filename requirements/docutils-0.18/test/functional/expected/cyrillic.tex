\documentclass[a4paper,russian]{article}
% generated by Docutils <http://docutils.sourceforge.net/>
\usepackage{cmap} % fix search and cut-and-paste in Acrobat
\usepackage{ifthen}
\usepackage[T1,T2A]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[english,russian]{babel}
\setcounter{secnumdepth}{0}

%%% Custom LaTeX preamble
% PDF Standard Fonts
\usepackage{mathptmx} % Times
\usepackage[scaled=.90]{helvet}
\usepackage{courier}

%%% User specified packages and stylesheets
\usepackage{cmlgc}

%%% Fallback definitions for Docutils-specific commands

% hyperlinks:
\ifthenelse{\isundefined{\hypersetup}}{
  \usepackage[colorlinks=true,linkcolor=blue,urlcolor=blue]{hyperref}
  \usepackage{bookmark}
  \urlstyle{same} % normal text font (alternatives: tt, rm, sf)
}{}
\hypersetup{
  pdflang={ru},
}

%%% Body
\begin{document}


\section{Заголовок%
  \label{section-1}%
}

первый пример: «Здравствуй, мир!»


\section{Title%
  \label{title}%
}

\foreignlanguage{english}{first example: “Hello world”.}


\section{Примечания%
  \label{section-2}%
}

Этот документ испытивает изображение LaTeXом кириллических букв.

\foreignlanguage{english}{This document tests the rendering of Latin and Cyrillic characters by the
LaTeX and XeTeX writers. Check the compiled PDF for garbage characters in
text and bookmarks.}

\end{document}
